pkgs/applications/editors/vscode/generic.nix

```diff
@@ -1,7 +1,7 @@
 { stdenv, lib, makeDesktopItem
 , unzip, libsecret, libXScrnSaver, wrapGAppsHook
 , gtk2, atomEnv, at-spi2-atk, autoPatchelfHook
-, systemd, fontconfig, libdbusmenu
+, systemd, fontconfig, libdbusmenu, buildFHSUserEnvBubblewrap
```

Comment:
Nit: `buildFHSUserEnvBubblewrap` is only meant as a temporary measure until it replaces `buildFHSUserEnv` completely, so vscode should probably not explicitly depend on it and rather override the `buildFHSUserEnv` with it in its callPackage.

Reply:
I was thinking about this, but I'm starting to like having dependency mapping in two places.